com.f3d-app.F3D.yaml

```
@@ -0,0 +1,307 @@
+app-id: com.f3d-app.F3D
```

```suggestion
app-id: com.f3d_app.F3D
```

It has to be with an underscore and not a `-`.